tests/test_slim_engine.py

```diff
@@ -1020,12 +1020,31 @@ def verify_genomic_elements(self, contig, ts):
         ge_types = self.slim_metadata_key0(
             ts.metadata["SLiM"]["user_metadata"], "genomicElementTypes"
         )
+        # getting the ids for the neutral mutation types
+        neutral_ids = [
+            mtdict["id"]
+            for mtdict in contig.mutation_types()
```

Comment:
Now that we put this information into metadata, we can (and I think *should* entirely remove this [`Contig.mutation_types()`](https://popsim-consortium.github.io/stdpopsim-docs/latest/api.html#stdpopsim.Contig.mutation_types) method - so, can we use something else here?